src/backend/access/external/pxffilters.c

```diff
@@ -406,6 +406,16 @@ pxf_serialize_filter_list(List *expressionItems)
 				appendStringInfo(resbuf, "%c%d", PXF_LOGICAL_OPERATOR_CODE, boolType);
 				break;
 			}
+			case T_NullTest:
+			{
+				elog(DEBUG1, "pxf_serialize_filter_list: node tag %d (T_NullTest)", tag);
+				NullTest *expr = (NullTest *) node;
+
+				/* filter expression for T_NullTest will not have any constant value */
+				PxfOperatorCode o = (expr->nulltesttype == IS_NULL) ? PXFOP_IS_NULL :  PXFOP_IS_NOTNULL;
```

Comment:
It would be safer to explicitly check whether expr->nulltesttype == IS_NULL or expr->nulltesttype == IS_NOTNULL instead of relying to default case. As an example in exising codebase: https://github.com/apache/incubator-hawq/blob/master/src/backend/executor/execQual.c#L3828